src/topology/algebra/ordered.lean

```diff
@@ -9,17 +9,25 @@ import data.set.intervals
 import topology.algebra.group
 import topology.constructions
 
-/-! # Theory of ordered topology
+/-! # Theory of topology on ordered spaces
 
 ## Main definitions
-`ordered_topology` and `orderable_topology`
 
-TODO expand
+The order topology on an ordered space is the topology generated by all open intervals (or
+equivalently by those of the form `(-∞, a)` and `(b, +∞)`). We introduce a mixin `order_topology α`
+saying that on the type α having a topological space structure and a preorder structure, the
+topological structure corresponds precisely to the order topology.
+
+We also introduce a mixin `order_closed_topology α` saying that the set of points `(x, y)` with
```

Comment:
I think "mixin" vocabulary is obscure unless we refer to a general explanation. I would explicitly say that `order_closed_topology α` lives in `Prop` and assumes `α` is already equipped with an order and a topology. The same comment applies elsewhere.

Reply:
Updated